docs/how-to/install-slice.rst

```diff
@@ -0,0 +1,96 @@
+How-to install from my own custom slice definitions?
+****************************************************
+
+First, you need to clone this repository:
+
+.. code-block:: sh
+
+    $ git clone https://github.com/canonical/chisel-releases/
+
+This repository is the database of slices definitions file. After that, you need to put your openssl.yaml file into the chisel-releases/slices folder.
+
+| Now we will install our openssl slice into our ROCK'S, so to initialize a rock you need to do:
+
+.. code-block:: sh
+
+    $ rockcraft init
+
+after this command your folder looks like this:
+
+.. code-block:: text
+
+    -chisel-releases/
+    -rockcraft.yaml
+
+Now you need to copy and paste this code into the rockcraft file:
+
+.. code-block:: yaml
+
+    name: ourslice
+    base: bare
+    build_base: "ubuntu:22.04"
+    version: '0.0.1'
+    summary: Install the slice
+    description: |
+        Slice install tutorial
+    license: GPL-3.0
+    platforms:
+        amd64:
+    parts:
+        build-context:
+            plugin: dump
+            source: chisel-releases/
+            source-type: local
+            organize:
+                '*': chisel-releases/
+
+        installslice:
+            plugin: go
+            source: https://github.com/canonical/chisel
+            source-type: git
+            build-snaps:
+                - go/1.17/stable
+            after: [build-context]
+            override-prime: $CRAFT_PART_INSTALL/bin/chisel cut --release chisel-releases/ --root . openssl_bins # name of your slice
+
+
+The build-context permits us to send our chisel-releases folder to the builder. The install slice section enables us to install our slice thanks to chisel into the ROCK.
+if you need more information about the yaml file, you can check the :ref:`reference` section.
+
+
+| To build your ROCK, you need to do the following:
+
+.. code-block:: sh
+
+    $ rockcraft
+
+To run it on docker, you need to convert the rocks into a docker image:
+
+.. code-block:: sh
+
+    $ skopeo --insecure-policy copy oci-archive:ourslice_0.0.1_amd64.rock docker-daemon:chisel-openssl:latest
+
+You can test it with the following:
+
+.. code-block:: sh
+    $ docker run chisel-openssl openssl
```

Comment:
```suggestion
    $ skopeo --insecure-policy copy oci-archive:ourslice_0.0.1_amd64.rock docker-daemon:chisel-openssl:latest
    $ docker run chisel-openssl openssl
```

Reply:
also, please paste the output of the commands and fix the oci archive filename once you change the YAML above